sbncode/Calibration/TrackCaloSkimmer_module.cc

```diff
@@ -289,7 +307,7 @@ void sbn::TrackCaloSkimmer::analyze(art::Event const& e)
     fWiresToSave.clear();
 
     // Fill the track!
-    FillTrack(*trkPtr, pfp, t0, trkHits, trkHitMetas, calo, rawdigits, track_infos);
+    FillTrack(*trkPtr, pfp, t0, trkHits, trkHitMetas, calo, rawdigits, track_infos, hasSBND, hasIcarus);
```

Comment:
Seems like it would be neater/clearer to have a single detector enum passed here, rather than two mutually exclusive booleans. Then all the `if(hasIcarus)` become `if(det == kICARUS)`

Reply:
I will get this fixed once I'm back from holiday (1st September). Thanks